src/components/RuntimeManager.js

```diff
@@ -347,7 +348,7 @@ export default class RuntimeManager extends PureComponent {
 
     const galaxyApp = getCurrentApp(tools.galaxy.appType)(apps)
 
-    return h(Fragment, [
+    return isAnalysisTabVisible() ? h(Fragment) : h(Fragment, [
```

Comment:
`null` would make the intention more explicit:

```suggestion
    return isAnalysisTabVisible() ? null : h(Fragment, [
```